test/conftest.py

```diff
@@ -90,16 +102,16 @@ def ipv6_san_server(tmp_path_factory):
 
 
 @pytest.fixture
-def stub_timezone(request):
+def stub_timezone(request: pytest.FixtureRequest) -> Generator[None, None, None]:
     """
     A pytest fixture that runs the test with a stub timezone.
     """
-    with stub_timezone_ctx(request.param):
+    with stub_timezone_ctx(request.param):  # type: ignore[attr-defined]
```

Comment:
`request` type should be `SubRequest` which is in `_pytest.fixtures.SubRequest` and does not exposed in `pytest`.
@bluetech Should I create a PR and add `SubRequest` in https://github.com/pytest-dev/pytest/blob/main/src/pytest/__init__.py?

Reply:
We have an existing issue for this: https://github.com/pytest-dev/pytest/issues/8763

For now I think what you did is best :+1: